iac-extensions/arm/src/test/resources/checks/LogRetentionCheck/Microsoft.Network_networkWatchers_flowLogs.bicep

```diff
@@ -0,0 +1,125 @@
+resource flowLogs1 'Microsoft.Network/networkWatchers/flowLogs@2022-07-01' = {
+  name: 'Noncompliant: Raise Issue as days is too low /'
```

Comment:
Nitpick: just wondering why there is a slash at the end of the name?

Reply:
I will remove it.